### PR TITLE
[TACHYON-1186] Move exception messages in TachyonConf.java to ExceptionMessages

### PR DESCRIPTION
**Pull request by sisirkoppaka**
Creates new ExceptionMessages for TachyonConf.java and replaces existing exception strings with ExceptionMessage-equivalents.

### CODE REVIEW

**Comment by haoyuan** [comment]
Jenkins, test this please.

File: common/src/main/java/tachyon/exception/ExceptionMessage.java [hunk 1]
```diff
@@ -111,6 +111,14 @@
   LINEAGE_INPUT_FILE_NOT_EXIST("The lineage input file {0} does not exist"),
   LINEAGE_OUTPUT_FILE_NOT_EXIST("No lineage has output file {0}"),
 
+  // Tachyon Conf
+  INVALID_CONFIGURATION_KEY("Invalid configuration key {0}"),
+  KEY_NOT_INTEGER("Configuration cannot evaluate key {0} as integer"),
+  KEY_NOT_DOUBLE("Configuration cannot evaluate key {0} as double"),
+  KEY_NOT_BYTES("Configuration cannot evaluate key {0} as bytes"),
+  DEFAULT_PROPERTIES_FILE_DOES_NOT_EXIST("The default Tachyon properties file does not exist"),
```
Comment (yupeng9) [inline common/src/main/java/tachyon/exception/ExceptionMessage.java]: Could you sort this alphabetically?

**Comment by yupeng9** [comment]
Thanks for extracting the exception messages into constants. I left one minor comment.

File: common/src/main/java/tachyon/exception/ExceptionMessage.java (hunk 1 above)
Comment (sisirkoppaka) [inline common/src/main/java/tachyon/exception/ExceptionMessage.java]: Done, and pushed. Thanks!

**Comment by sisirkoppaka** [comment]
@yupeng9 Thanks for the note, sorry I forgot. Updated and pushed.

**Comment by yupeng9** [comment]
LGTM

**Comment by gpang** [comment]
ok to test

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**